src/csharp/Microsoft.Spark.UnitTest/Microsoft.Spark.UnitTest.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.1' ">
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp3.0' ">
```

Comment:
This can just get removed. There is never a case where TargetFramework != netcoreapp3.0.